src/repo/learnocaml_process_tutorial_repository.ml

```diff
@@ -63,6 +63,11 @@ let main dest_dir =
     match !tutorials_index with
     | Some tutorials_index -> tutorials_index
     | None -> !tutorials_dir / "index.json" in
+  let tutorials_path =
```

Comment:
This duplication frightens me.

Reply:
You mean `tutorials_dir` vs `tutorials_path`? It's poor naming actually... The global reference `!tutorials_dir` is the configuration of the tutorial _source_ directory, while `tutorials_path` points to the destination directory under `www`. I'll change for more explicit names.